### DISCUSSION TITLE
DogeCoin Core wallet shows 0 coins.

### DISCUSSION THREAD

**Discussion by a2logistics** [dogecoin/dogecoin#3733] [Q&A]
i Purchased some dogecoins in 2017. and transferred my coins to Dogecon Core.  I still have the wallet on my desktop. i just opened after so many years, first it took few days to sync with network. and now when its completed it shows 0 Coins. i did not save my private keys anywhere.  What should i do in this situation.  is there any way to recover the private keys from the dat. file i have in my C drive for this wallet.    or something else i need to do on the wallet, such as update etc

**Comment by Duke261** [comment]
Hello @a2logistics it's a scam don’t click on any link. @Jarrel-doe have been reported in multiple repositories. The scammers post phishing links. Admin please ban or find this scammer.

**Comment by rishabhsingroha** [comment]
hahah domed

**Comment by G-Titan** [comment]
Lost coins

But guys on a serious note I've been digging and I am not sure what I am looking at. I am tired of thinking or working on this alone so anyone explain to me what is going on here?
<img width="910" height="383" alt="image" src="https://github.com/user-attachments/assets/9c4a4559-8bca-4f44-8b57-9f0a87a0aa53" />


tried running a full node, tried mining but it is so damn competitive!
cut a brother some slack and share some doge okay?

Here's my address & thank you for the cup of coffee: DMzsThhJ327VqXoaEsCr2gXrofTREhsYt1 

(it would be a teaser in my country if I got 2million doge) just saying

**Comment by VishalRaut2106** [comment]
I understand this is frustrating. The "LoadBlockIndexDB: transaction index disabled" error is significant - it suggests your wallet database may have become corrupted or incompletely synchronized. Here are some steps that might help:

**Immediate Troubleshooting:**
1. **Backup first** - Make a copy of your wallet.dat file before trying anything else. It's located in your Dogecoin data directory (~/.dogecoin/ on Linux/Mac, C:\\Users\\YourName\\AppData\\Roaming\\Dogecoin\\ on Windows).

2. **Rescan blockchain** - Start DogeCoin Core with the `-rescan` flag to rebuild the wallet database:
   - Windows: Create a shortcut to dogecoin-qt.exe and add `-rescan` to the Target field
   - Linux/Mac: Run `./dogecoin-qt -rescan` from terminal
   This process can take a while but may recover your balance.

3. **Check transaction index** - The error suggests the transaction index is disabled. Try deleting the block index:
   - Close DogeCoin Core completely
   - Delete the `blocks` folder in your Dogecoin data directory
   - Restart DogeCoin Core to redownload and rebuild the index

**Important considerations:**
- Since you don't have your private keys saved, recovery depends on the wallet file itself being intact
- If these steps don't work, the wallet data may be irretrievably lost
- You might consider posting on r/dogecoin or the Dogecoin forum where users with more experience in wallet recovery may be able to help

Good luck! Let me know if these steps help or if you need clarification on any of them.
